### PR TITLE
Dev add sentry playground

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Changes

- added sentry to playground

### Related Issues

#3101 

### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist
- [x] Screenshots/Video attached (needed for UI changes)
image

### CODE REVIEW

File: packages/playground/src/main.ts [hunk 1]
```diff
@@ -22,6 +23,19 @@ app.config.errorHandler = error => {
   );
 };
 
+Sentry.init({
+  app,
+  dsn: "https://b9af6796f176d1f02837a06f0da3caee@dev.sentry.grid.tf/2",
```
Comment (xmonader) [inline packages/playground/src/main.ts]: Why is this not coming from the process env ?

File: packages/playground/src/main.ts [hunk 2]
```diff
@@ -22,6 +23,19 @@ app.config.errorHandler = error => {
   );
 };
 
+Sentry.init({
```
Comment (xmonader) [inline packages/playground/src/main.ts]: How does it behave if no DSN configured?

Comment (amiraabouhadid) [reply packages/playground/src/main.ts]: no log errors in playground and no syncing with sentry
image

**Review by Mahmoud-Emad — APPROVED** [review]
LGTM

**Review by zaelgohary — APPROVED** [review]

**Pull request merged**